build-parent/pom.xml

```diff
@@ -67,6 +67,7 @@
         <version.awaitility>4.0.3</version.awaitility>
         <version.plexus.util>3.5.1</version.plexus.util>
         <version.jackson.mapper.asl>1.9.3</version.jackson.mapper.asl>
+        <version.kafka.client>3.8.0</version.kafka.client>
```

Comment:
Q: Do we need to also update the kafka chart version, or is the client compatible?

Reply:
Carter was able to test wit 3.8 and it is compatible. However I noticed data-access module doesnt need to use this version and exclude the quarkus version any more. So will remove it here and remove the exclusion from the quarkus dependency

Reply:
Thinking we should add something to the release notes about how that exclusion is no longer necessary

Reply:
Added a blurb in the release notes